### DISCUSSION TITLE
Deprecated data in docs

### DISCUSSION THREAD

**Discussion by sovyk-v** [fastapi/fastapi#9901] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Here is a code snippet from docs:

@app.patch("/items/{item_id}", response_model=Item)
async def update_item(item_id: str, item: Item):
    stored_item_data = items[item_id]
    stored_item_model = Item(**stored_item_data)
    update_data = item.dict(exclude_unset=True)
    updated_item = stored_item_model.copy(update=update_data)
    items[item_id] = jsonable_encoder(updated_item)
    return updated_item
```

### Description

There is a deprecated data in docs. In docs they say to use .dict to transform pydantic model into dictionaty, but it's deprecated method, pydantic offers .model_dump for it now. 

https://fastapi.tiangolo.com/tutorial/body-updates/

### Operating System

Linux

### FastAPI Version

0. 100. 0

### Python Version

3.10.6

**Comment by sovyk-v** [comment]
The same with .copy
Now it's recommended to use .model_copy by pydantic

**Comment by YuriiMotov** [comment]
Thanks for reporting it!

Duplicate with the answer: https://github.com/fastapi/fastapi/discussions/11912#top
